### PR TITLE
[lint-unknown-licenses] Verify unknown licenses in lint-licenses task

### PR DESCRIPTION
**Pull request by CelianR**
### What does this PR do?

Unknown licenses were checked only in `generate-licenses`, now it is checked also in `lint-licenses`.

### Describe how to test/QA your changes

I modified `.copyright-overrides.yml` and `lint-licenses` exited with a non-zero exit code as expected

### CODE REVIEW

File: tasks/licenses.py [hunk 1]
```diff
@@ -72,11 +73,39 @@ def get_licenses_list(ctx):
...
+def _verify_unknown_licenses(licenses, licenses_filename):
+    """
+    Check that all deps have a non-"UNKNOWN" copyright and license
+    """
+    unknown_licenses = False
+    for line in licenses:
+        parts = [part.strip().casefold() for part in line.split(',')]
+        if 'unknown' in parts:
+            unknown_licenses = True
+            print(f"! {line}")
```
Comment (amenasria) [inline tasks/licenses.py:93]: **❓ question:** ‏Was the `print` left on purpose ? If it wasn't:
```suggestion
        unknown_licenses = unknown_licenses and 'unknown' in parts
```

File: tasks/licenses.py [hunk 2]
```diff
@@ -72,11 +73,39 @@ def get_licenses_list(ctx):
...
+    if unknown_licenses:
+        raise Exit(
+            message=textwrap.dedent(
+                """\
+                At least one dependency's license or copyright could not be determined.
+
+                Consult the dependency's source, update
+                `.copyright-overrides.yml` or `.wwhrd.yml` accordingly, and run
+                `inv generate-licenses` to update {}."""
+            ).format(licenses_filename),
```
Comment (amenasria) [inline tasks/licenses.py:104]:
```suggestion
                f"""\
                At least one dependency's license or copyright could not be determined.

                Consult the dependency's source, update
                `.copyright-overrides.yml` or `.wwhrd.yml` accordingly, and run
                `inv generate-licenses` to update {licenses_filename}."""
            ),
```

**Review by amenasria — APPROVED** [review]
A few comments otherwise LGTM !

**Review by KevinFairise2 — APPROVED** [review]

File: tasks/licenses.py (hunk 1 above)
Comment (CelianR) [reply tasks/licenses.py:93]: This print was already there before, it is useful to print each unknown license (if multiple). Imo it is better to leave it here

**Pull request merged**